mqtt 3/1.0.0/modules/ROOT/pages/edits/mqtt3-connector-quality-of-service.adoc

```diff
@@ -0,0 +1,25 @@
+= Configure the Quality of Service for MQTT 3 Connector
+
+The MQTT protocol enables you to specify levels of delivery assurance for inbound or outbound messages. These levels
```

Comment:
In reading this, I wonder if we should distinguish more clearly among the verbs...:

- send
- deliver: this could mean either sent or received
- receive

Reply:
Just to consider...:
```suggestion
Using the MQTT protocol, configure a level of message-delivery assurance (quality of service, or QoS): 
```